src/services/caching/__mocks__/service.ts

```diff
@@ -28,4 +28,8 @@ export class CachingService implements ICachingService {
   async delete(filePath: string) {
     this.cache.delete(filePath);
   }
+
+  async deleteMany(filePaths: string[]): Promise<void> {
+    filePaths.forEach((filepath) => this.cache.delete(filepath));
+  }
```

Comment:
Is `this.cache.delete` synchronous? Do we need it to be `async`? Or should we wait somewhere?